test/DurableTask.SqlServer.Tests/Integration/DatabaseManagement.cs

```diff
@@ -127,7 +128,61 @@ public async Task CanCreateAndDropSchema(bool isDatabaseMissing)
             // We know that all objects were successfully removed if the "dt" no longer exists.
             Assert.DoesNotContain("dt", testDb.GetSchemas());
         }
+        
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task CanCreateAndDropSchemaWithCustomSchemaName(bool isDatabaseMissing)
+        {
```

Comment:
We should have a test that checks no object gets created under dt schema, in case it is overriden by another name.

Reply:
On line 183 there is an assert checking there no "dt" schema and such, no objects under it. Is this what you were referring to?